src/reactions/aggregateReactions.js

```diff
@@ -1,36 +1,52 @@
 /* @flow strict-local */
-import type { Reaction, AggregatedReaction } from '../types';
+import type { Reaction, AggregatedReaction, User } from '../types';
 
 export default (
   reactions: $ReadOnlyArray<Reaction>,
-  ownEmail: string,
-): $ReadOnlyArray<AggregatedReaction> => {
-  const reactionMap = new Map();
-  reactions.forEach(x => {
-    if (!reactionMap.has(x.emoji_name)) {
-      reactionMap.set(x.emoji_name, {
-        code: x.emoji_code,
-        count: 1,
-        name: x.emoji_name,
-        type: x.reaction_type,
-        selfReacted: false,
-      });
-    } else {
-      const prevReaction = reactionMap.get(x.emoji_name);
-      if (prevReaction) {
-        reactionMap.set(x.emoji_name, {
-          ...prevReaction,
-          count: prevReaction.count + 1,
-        });
-      }
-    }
+  ownUserId: number,
+  users?: Map<number, User>,
```

Comment:
Hmm, let's see.

It looks like what this `users` parameter is being used for is just for `users.get(x.user.user_id)`, and as a sort of flag by testing `users` for truthiness.

How about simply skipping that and letting the `AggregatedReaction` have an array of user IDs rather than user objects? Then the parameter can go away entirely, and the consumer of the data can look up for itself the user IDs as needed.